skimage/transform/tests/test_histogram_matching.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+
+from skimage.transform import histogram_matching
+from skimage import transform
+from skimage import data
+
+from skimage._shared.testing import assert_array_almost_equal, \
+    assert_almost_equal
+
+import pytest
+
+
+@pytest.mark.parametrize('array, template, expected_array', [
+    (np.arange(10), np.arange(100), np.arange(9, 100, 10)),
+    (np.random.rand(4), np.ones(3), np.ones(4))
+])
+def test_match_array_values(array, template, expected_array):
+    # when
+    matched = histogram_matching._match_cumulative_cdf(array, template)
+
+    # then
+    assert_array_almost_equal(matched, expected_array)
+
+
+class TestMatchHistogram:
```

Comment:
I don't understand why this needs to be a class instead of a bunch of functions, like we do everywhere else in the library